tools/monitoring/config/dashboards/definations/inspektor-gadget.json

```diff
@@ -83,7 +83,7 @@
             "uid": "PBFA97CFB590B2093"
           },
           "editorMode": "code",
-          "expr": "sum by(le) (dns_latency_bucket)",
+          "expr": "sum by(le) (dns_latency_nanoseconds_bucket)",
```

Comment:
We were using [nanoseconds](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/tools/monitoring/config/histogram.yaml#L12) before as well, it is just that since the exporter now added it as a known unit the resulting metrics will include[ _nanoseconds](https://github.com/open-telemetry/opentelemetry-go/pull/4374/files#diff-9799d8f4e9503511186103b23b6d230e6aa6037ede5b6341b8f3e47017cc789cR340). So just a cosmetic change.